api/v1alpha1/frontend_types.go

```diff
@@ -148,6 +149,12 @@ func TruePtr() *bool {
 	return &t
 }
 
+// TruePtr returns a pointer to True
```

Comment:
I noticed that the TruePtr function has a comment that says `TruePtr returns a pointer to True` as well. I'm assuming you wanted to update this to reflect FalsePtr?

Reply:
I WANT MY COMMENTS TO BE LIES SO PEOPLE WILL PAY ATTENTION